platooning_strategic/src/platoon_strategic.cpp

```diff
@@ -696,7 +739,15 @@ namespace platoon_strategic
                     waitingStartTime = ros::Time::now().toNSec()/1000000;
                     lw_applicantId_ = msg.header.sender_id;
                     return MobilityRequestResponse::ACK;
-                } else {
+                } 
+                else if(isDistanceCloseEnough && !laneConditionsSatisfied) {
```

Comment:
Minor formatting quibble, let's move this else-if up to the line with the brace `} else if ... {`

Reply:
Fixed.